packages/repository-modernizer/src/restructure-filters.js

```diff
@@ -64,6 +64,7 @@ var RestructureFilterPaths = {
                         ? project.relativePathToExistingFilterXml
                         : constants.FILTER_PATH
                 );
+                if(!project.relativePathToExistingFilterXml.endsWith("pom.xml")){
```

Comment:
Please fix the linting issues. Did you skip the eslint check in commit (which would have caught these issues)?